src/main/java/dinkplugin/DinkPluginConfig.java

```diff
@@ -764,14 +764,38 @@ default boolean lootIncludeClueScrolls() {
         return true;
     }
 
+    @ConfigItem(
+        keyName = "lootItemAllowlist",
+        name = "Item Allowlist",
+        description = "Always fire notifications for these items, despite value settings. " +
+            "(One name per line, case-insensitive)",
```

Comment:
Put the "One name per line..." on a new line - it doesn't fit nicely on a single line
It can probably have its parentheses removed too if it's on a new line
Same for the deny list
With the extra space, maybe change it to "...for these item names..."
"One item name per line...", or only changing the lower line to "One item per line..."

![image](https://github.com/pajlads/DinkPlugin/assets/962989/d8fe212c-27af-4e6f-bbfa-429d7b0d42a5)